python/looker_sdk/sdk/api40/methods.py

```diff
@@ -8442,8 +8442,6 @@ def create_user(
         transport_options: Optional[transport.PTransportSettings] = None,
     ) -> models.User:
         """Create User"""
-        if body:
-            assert isinstance(body, models.WriteUser)
```

Comment:
I expected a bigger delta than this, for sure. Didn't you remove these assertions from all methods? Or was this the only one remaining that needed to be removed? Or was this added by hand then removed when you determined it wasn't needed for generated code?

Reply:
oops,  my bad - I need to update codegen to remove them all, good catch!